dev/com.ibm.ws.microprofile.reactive.streams.operators_fat/fat/src/com/ibm/ws/microprofile/reactive/streams/test/suite/FATSuite.java

```diff
@@ -30,30 +31,38 @@
                 ReactiveConcurrentWorkTest.class
 })
 public class FATSuite {
-    public static final String MPRS10_ID = MicroProfileActions.MP21_ID + "_" + "MPRS10";
-    public static final String MPRS30_MP60_ID = MicroProfileActions.MP60_ID + "_" + "MPRS30";
+    public static final String MP_REACTIVE_STREAMS_10 = "mpReactiveStreams-1.0";
+    public static final String MP_REACTIVE_STREAMS_30 = "mpReactiveStreams-3.0";
 
-    public static final FeatureSet MPRS10 = MicroProfileActions.MP21.addFeature("mpReactiveStreams-1.0").build(MPRS10_ID);
-    public static final FeatureSet MPRS30_MP60 = MicroProfileActions.MP60.addFeature("mpReactiveStreams-3.0").build(MPRS30_MP60_ID);
+    public static final String MP21_RSO10_ID = MicroProfileActions.MP21_ID + "_" + "mpRSO10";
+    public static final String MP50_RSO30_ID = MicroProfileActions.MP50_ID + "_" + "mpRSO30";
+    public static final String MP60_RSO30_ID = MicroProfileActions.MP60_ID + "_" + "mpRSO30";
+    public static final String MP61_RSO30_ID = MicroProfileActions.MP61_ID + "_" + "mpRSO30";
```

Comment:
```suggestion
    public static final String MP21_RS10_ID = MicroProfileActions.MP21_ID + "_RS10";
    public static final String MP50_RS30_ID = MicroProfileActions.MP50_ID + "_RS30";
    public static final String MP60_RS30_ID = MicroProfileActions.MP60_ID + "_RS30";
    public static final String MP61_RS30_ID = MicroProfileActions.MP61_ID + "_RS30";
```

This lines up with the naming that was do for ReactiveMessaging